update/updateDb.v3.5.sql

```diff
@@ -21,6 +21,8 @@ ENGINE = InnoDB;
 
 ALTER TABLE `encoder_queue` ADD COLUMN `override_status` VARCHAR(45) NULL;
 
+ALTER TABLE `encoder_queue` ADD COLUMN `worker_ppid` INT NULL;
```

Comment:
This should be version 3.6 otherwise who already is on 3.5 will not be able to follow it.
I will fix it for you